### PR TITLE
readded depression's page

### PR DESCRIPTION
**Pull request by A-UNDERSCORE-D**

### CODE REVIEW

File: content/community/depression.md [hunk 1]
```diff
@@ -6,3 +6,130 @@ draft: false
...
+
+## Channel Operators {#ops}
+### Channel Owner: 
+- [Duck](https://snoonet.org/duck)
+
+### Channel Admin: 
+- Caeli
+
+### Channel Staff: 
+- [A_D](https://snoonet.org/A_D)
```
Comment (linuxdaemon) [inline content/community/depression.md]:
```suggestion
- [A_D](/staff/a_d)
```

File: content/community/depression.md [hunk 2]
```diff
@@ -6,3 +6,130 @@ draft: false
...
+- AHare
+- GhostyOne
+- jglauche
+- [Madbrad200](https://snoonet.org/Madbrad200)
```
Comment (linuxdaemon) [inline content/community/depression.md]:
```suggestion
- [Madbrad200](/staff/madbrad200)
```

File: content/community/depression.md [hunk 3]
```diff
@@ -6,3 +6,130 @@ draft: false
...
+- MuffinMedic
+- TheStrangeOne
+- toomin
+- [void](https://snoonet.org/void)
```
Comment (linuxdaemon) [inline content/community/depression.md]:
```suggestion
- [void](/staff/void)
```

File: content/community/depression.md [hunk 4]
```diff
@@ -6,3 +6,130 @@ draft: false
...
+
+Our main function is to listen, be here for one another and offer peer support. Whether you’re just having a bad day, issues that are bringing you down, or currently suffering from one of the many variations of depression, we are here for you.
+
+Please understand that although we are here to provide peer support, our channel is not intended as a replacement for counselling from a qualified professional and as such, all information/advice provided by our users is to be considered anecdotal and does not carry any weight. Please note, **we are not affiliated with [/r/depression](https://reddit.com/r/depression)** – that said, they are a wonderful group and we encourage you to check them out.
+
+[back to top](top)
+
+## Channel Operators {#ops}
+### Channel Owner: 
+- [Duck](https://snoonet.org/duck)
```
Comment (linuxdaemon) [inline content/community/depression.md]:
```suggestion
- [Duck](/staff/duck)
```

File: content/community/depression.md [hunk 5]
```diff
@@ -6,3 +6,130 @@ draft: false
 type: community
 kind: "health_and_food"
 ---
+
+[Purpose](purpose)  ||  [Channel Operators](ops) ||  [Rules & Guidelines](guidelines)  ||  [Ban Appeals](appeals)  || [Webchat](https://webchat.snoonet.org/depression) || [Links & Resources](resources)
```
Comment (linuxdaemon) [inline content/community/depression.md]: I'm not 100% certain but I'm pretty sure `[Purpose](purpose)` would try to link to a page called `purpose` in the current directory, it needs to start with `#` to link to an ID

**Review by linuxdaemon — CHANGES_REQUESTED** [review]

File: content/community/depression.md (hunk 5 above)
Comment (A-UNDERSCORE-D) [reply content/community/depression.md]: The # is not required, I tested it

Comment (linuxdaemon) [reply content/community/depression.md]: I just tested it and yes it is, clicking "Purpose" navigates to `http://localhost:1313/community/depression/purpose`

Comment (A-UNDERSCORE-D) [reply content/community/depression.md]: Yeah I just did as well. Its SUPPOSED to work. Fixing.

**Review by n7st — APPROVED** [review]

**Review by linuxdaemon — APPROVED** [review]

**Pull request merged**